docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -489,6 +489,44 @@ file suffix.
         ``additional_certificates`` is a list of all other
         :class:`~cryptography.x509.Certificate` instances in the PKCS12 object.
 
+.. function:: generate_pkcs12(password, name, key, cert, ca, backend,
+                    key_encryption_algorithm,
+                    cert_encryption_algorithm,
+                    encryption_algorithm_iter_count,
+                    mac_iter_count, key_type)
```

Comment:
Is it practical to have only a single encryption_algorithm parameter to start with?

Reply:
The default encryption algorithms for key and cert are different:
```
    /* Set defaults */
    if (!nid_cert)
#ifdef OPENSSL_NO_RC2
        nid_cert = NID_pbe_WithSHA1And3_Key_TripleDES_CBC;
#else
        nid_cert = NID_pbe_WithSHA1And40BitRC2_CBC;
#endif
    if (!nid_key)
        nid_key = NID_pbe_WithSHA1And3_Key_TripleDES_CBC;
```
I assumed that means that the user might want to have different algorithms for key and cert.
It is a product question, and since I am not proficient enough with this package, I let you decide.

Reply:
Then I'm a vote for just having one option for now. (And holy crap, it defaults to RC2 if OpenSSL is compiled with it....)